translator/translate/otel/processor/appsignalsmetricstransformprocessor/config/appSignalsRuntimeConfig.yaml

```diff
@@ -0,0 +1,279 @@
+transforms:
+  - include: jvm.cpu.recent_utilization
+    action: update
+    new_name: JVMCpuRecentUtilization
+    operations:
+      - action: aggregate_labels
+        label_set: []
+        aggregation_type: sum
+      - action: add_label
+        new_label: Telemetry.Source
```

Comment:
Wouldnt it be cleaner to set these on the java agent when exporting them?

Reply:
The attribute is added so the metrics can be selected by the EMF exporter to generate metrics.
Ref:
https://github.com/aws/amazon-cloudwatch-agent/blob/bde3bd9775ae1d4e4f8a2fdb92d7b6fdd5186fba/translator/tocwconfig/sampleConfig/base_appsignals_config.yaml#L23-L25